lib/fs-loader.js

```diff
@@ -132,13 +132,20 @@ const load = async (dir, options, config) => {
   if (!Array.isArray(dir)) {
     dir = [dir];
   }
+  const newOptions = Object.assign({}, options);
```

Comment:
```suggestion
  const newOptions = common.clone(options);
```

Reply:
```suggestion
  const options = { ...options };
```
?

Also, why is there a need to make copy?